### ISSUE TITLE
TODO: `new NativeEventEmitter()` warning when  ScreenGuardModule.register is called

### ISSUE THREAD

**Issue by alexbennycodes** [gbumps/react-native-screenguard#36]
WARN  `new NativeEventEmitter()` was called with a non-null argument without the required `removeListeners` method. keeps coming up when ScreenGuardModule.register() is called

**Comment by gbumps** [comment]
Thanks for reporting. Seems not too urgent for me to make a fix right now, so this warning will be fixed with upcoming version.

**Comment by gbumps** [comment]
All contribution are welcome

**Comment by gbumps** [comment]
hi @alexbennycodes, I've updated this on branch release/0.3.4, please check as following:

inside `package.json`, find `react-native-screenguard` and replace with this line

```
    "react-native-screenguard": "git+https://github.com/gbumps/react-native-screenguard.git#release/0.3.4",
```
please check and let me know if you need anything.

Thanks!

**Comment by gbumps** [comment]
If nothing change, I'm gonna release this version.

**Comment by alexbennycodes** [comment]
@gbumps thanks for your help

**Comment by gbumps** [comment]
This version has been released at `v0.3.5`.
